gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java

```diff
@@ -864,4 +874,8 @@ public void dispose() {
         taskItems.dispose();
         closeViews();
     }
+
+    public boolean canBeClosed() {
```

Comment:
Same remark

Reply:
ProjectPane extends Tab which already has a final isClosable() method. I did not rename it here.